src/Compilers/VisualBasic/Portable/Semantics/Conversions.vb

```diff
@@ -1210,8 +1211,29 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
         End Function
 
         Public Shared Function ClassifyTupleConversion(source As BoundTupleLiteral, destination As TypeSymbol, binder As Binder, <[In], Out> ByRef useSiteDiagnostics As HashSet(Of DiagnosticInfo)) As ConversionKind
+            If source.Type = destination Then
```

Comment:
>source.Type = destination [](start = 15, length = 25)

Symbol comparison shouldn't be used to detect Identity conversion. IsSameTypeIgnoringCustomModifiers helper should be used instead.